packages/camera/camera/ios/Classes/FLTSavePhotoDelegate.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
This is now exposed in the plugin API, I don't think we want that.  I think keep most of this in the implementation and only expose `FLTSavePhotoDelegate_Test.h` in the test module.

Reply:
Oh I didn't add it to the umbrella header so it's not exposed outside of this package.

Reply:
Also removed thread safe wrappers from the umbrella header

Reply:
Oh right right I forgot this uses a real umbrella header and not one generated by CocoaPods.

I think you're still exposing it in the test module since you're `#import`ing it in the `_Test` header.

Reply:
yep

Reply:
The original header and the test header are both exposed for unit test only